packages/uploadthing/src/next-legacy.ts

```diff
@@ -38,6 +38,7 @@ export const createRouteHandler = <TRouter extends FileRouter>(
     for (const [name, value] of response.headers) {
       res.setHeader(name, value);
     }
-    return res.json(response.body);
+    // FIXME: Should be able to just forward it instead of consuming it first
```

Comment:
Would be nice, but iI don't think this is a huge deal.

Reply:
nah lets just ship as-is and see if we ever find it high priority enough to optimize a little bit... the json blobs are never huge anyways so it's not like we're gonna load a bunch of stuff into memory